src/acl/Ip.cc

```diff
@@ -438,8 +472,7 @@ ACLIP::parse()
             /* pop each result off the list and add it to the data tree individually */
             acl_ip_data *next_node = q->next;
             q->next = nullptr;
-            if (!data->find(q,acl_ip_data::NetworkCompare))
-                data->insert(q, acl_ip_data::NetworkCompare);
+            Acl::SplayInserter<acl_ip_data*>::Merge(*data, std::move(q));
```

Comment:
Since q is a pointer - can we just pass it by value instead of this complexity with std::move()?